dataScrape/src/apiPull.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+class apiPull
+{
+    private $pokemonDataList;
+
+    /**
+     * apiPull constructor.  Pulls each pokemon JSON from api, and appends relevant data to PokemonDataList
+     */
+    public function __construct()
+    {
+        for ($i = 1; $i <= 151; $i++)
+        {
+            $curl = curl_init();
+
+            curl_setopt_array($curl, array(
+                CURLOPT_URL => "https://pokeapi.co/api/v2/pokemon/" . $i . '/',
+                CURLOPT_RETURNTRANSFER => true,
+                CURLOPT_TIMEOUT => 30,
+                CURLOPT_HTTP_VERSION => CURL_HTTP_VERSION_1_1,
+                CURLOPT_CUSTOMREQUEST => "GET",
+                CURLOPT_HTTPHEADER => array(
+                    "cache-control: no-cache"
+                ),
+            ));
+
+            $response = curl_exec($curl);
+
+            curl_close($curl);
+
+            $pokemon = json_decode($response);
```

Comment:
Is there a way of just getting back the information we need, and not all the information available from the API? If there isn't don't worry about it.

Reply:
Not possible without doing multiple requests per pokemon, which would hit the api request limit too soon.